andela/suyabay
# Align episode action icon to bottom right of the parent div.

#### What does this PR do?
- Align episode action icons to the bottom right of the parent div
#### Description of Task to be completed?
- Add a class name `right` to float the div to the right
- Add `margin-top : 10px` to the action icon div to align properly to the bottom of the parent div
  #### How should this be manually tested?
-  Visit `http://suyabay-staging.herokuapp.com` you should see the episode action icons floated to the right and well aligned to the parent div
#### Any background context you want to provide?
#### What are the relevant pivotal tracker stories?

 121024353
#### Screenshots (if appropriate)
#### Questions:

[@unicodeveloper](https://github.com/unicodeveloper)
[@andela-oadebayo](https://github.com/andela-oadebayo)
